components/ai_chat/ai_chat_tab_helper.cc

```diff
@@ -288,24 +289,30 @@ void AIChatTabHelper::CleanUp() {
 
 void AIChatTabHelper::MakeAPIRequestWithConversationHistoryUpdate(
     const ConversationTurn& turn) {
-  AddToConversationHistory(turn);
+  std::string prompt = base::ReplaceStringPlaceholders(
+      l10n_util::GetStringUTF8(IDS_AI_CHAT_SUMMARIZE_PROMPT),
+      {article_text_, GetConversationHistoryString(), turn.text}, nullptr);
 
   std::string prompt_with_history =
-      base::StrCat({GetConversationHistoryString(), ai_chat::kAIPrompt});
+      base::StrCat({prompt, ai_chat::kAIPrompt, " <response>\n"});
+
+  if (turn.visibility != ConversationTurnVisibility::DONT_ADD) {
```

Comment:
Did you forget to commit the .mojom file? This branch doesn't have the `DONT_ADD` member.

Reply:
forgot to push 👍